utilities/rocAL/rocAL_basic_test/rocAL_basic_test.cpp

```diff
@@ -50,7 +50,7 @@ int main(int argc, const char ** argv)
     // check command-line usage
     const int MIN_ARG_COUNT = 2;
     if(argc < MIN_ARG_COUNT) {
-        std::cout <<  "Usage: image_augmentation <image_dataset_folder> <label_text_file_path> <test_case:0/1> <processing_device=1/cpu=0>  decode_width decode_height <gray_scale:0/rgb:1> decode_shard_counts \n";
+        std::cout <<  "Usage: ro_basic_test <image_dataset_folder> <label_text_file_path> <test_case:0/1> <processing_device=1/cpu=0>  decode_width decode_height <gray_scale:0/rgb:1> decode_shard_counts \n";
```

Comment:
@rrawther did you mean ./rocal_basic_test in usage

Reply:
fixed. I have forgotten to push renamed file